CHANGELOG.rst

```diff
@@ -1,6 +1,31 @@
 template formula

```

Comment:
Excellent, tagging releases has already started!  May I recommend standardising our changelogs according to the format set out at https://keepachangelog.com/en/1.0.0/?

Reply:
I pushed the tags myself. :)

Reply:
thats on the todo list...I hope....

Reply:
@aboe76 Yes, I've already pulled and appreciated the tags in `gitk`!  Looks great, thanks for running with that.